__tests__/storage.test.js

```diff
@@ -291,4 +292,125 @@ describe('Tozny', () => {
+      writerClient,
+      created.group.groupID,
+      groupMembersToAdd
+    )
+    let clientID = []
+    clientID.push(readerClient.clientId)
+    clientID.push(readerClient.clientId)
+    const removeGroupMembers = await ops.removeGroupMembers(
+      writerClient,
+      created.group.groupID,
+      clientID
+    )
+    expect(removeGroupMembers).toBe(true)
+  })
+  it('can list group members from a group', async () => {
+    const groupName = `testGroup-${uuidv4()}`
+    const created = await ops.createGroup(writerClient, groupName)
+    const groupMember = new GroupMember(readerClient.clientId, { read: true })
+    let groupMembersToAdd = []
+    groupMembersToAdd.push(groupMember)
+    let addMembers = await ops.addGroupMembers(
+      writerClient,
+      created.group.groupID,
+      groupMembersToAdd
+    )
+    const groupMemberList = await ops.listGroupMembers(
+      writerClient,
+      created.group.groupID
+    )
+    let groupMemberListExpected = [
+      {
+        client_ids: writerClient.clientId,
+        capability_names: ['MANAGE_MEMBERSHIP'],
+      },
+      {
+        client_ids: readerClient.clientId,
+        membership_key: addMembers[0].membership_key,
+        capability_names: ['READ_CONTENT'],
+      },
+    ]
+    expect(groupMemberList).toMatchObject(groupMemberListExpected)
+  })
+  it('can list records shared with groups', async () => {
+    const groupName = `testGroup-${uuidv4()}`
+    const created = await ops.createGroup(writerClient, groupName)
+    const groupMember = new GroupMember(readerClient.clientId, { read: true })
+    let groupMembersToAdd = []
+    groupMembersToAdd.push(groupMember)
+    await ops.addGroupMembers(
+      writerClient,
+      created.group.groupID,
+      groupMembersToAdd
+    )
+    await ops.listGroupMembers(writerClient, created.group.groupID)
+    let sharedWithGroup = await ops.listRecordsSharedWithGroup(
+      readerClient,
+      created.group.groupID,
+      [],
+      0,
+      10
+    )
+    let sharedWithGroupExpected = []
+    expect(sharedWithGroup).toMatchObject(sharedWithGroupExpected)
+  })
+  it('can share records with a group', async () => {
+    const groupName = `testGroup-${uuidv4()}`
+    const created = await ops.createGroup(writerClient, groupName)
+    const groupMember = new GroupMember(readerClient.clientId, {
+      read: true,
+      share: true,
+    })
+    let groupMembersToAdd = []
+    groupMembersToAdd.push(groupMember)
+    await ops.addGroupMembers(
+      writerClient,
+      created.group.groupID,
+      groupMembersToAdd
+    )
+    const type = 'say-hello'
+    const data = { hello: 'world' }
+    const meta = { hola: 'mundo' }
+    await ops.writeRecord(readerClient, type, data, meta)
+    let shareWithGroup = await ops.shareRecordWithGroup(
+      readerClient,
+      created.group.groupID,
+      type
+    )
+    let recordExpected = {
+      record_type: type,
+    }
+    expect(shareWithGroup).toMatchObject(recordExpected)
```

Comment:
The share operation was successful, but how do we know that records are actually shared without listing records?